packages/containers-shared/src/utils.ts

```diff
@@ -29,7 +29,51 @@ export const runDockerCmd = async (
 	});
 };
 
-export const verifyDockerInstalled = async (dockerPath: string) => {
+export const runDockerCmdWithOutput = async (
```

Comment:
Might be usefull to add when the returned promise resolves with what, same for reject